File: src/main/java/freemusic/music/pojo/wy/NeteaseLostSong.java
package freemusic.music.pojo.wy;

import java.util.List;

/**
 * Created by qtfreet on 2017/2/6.
 */
public class NeteaseLostSong {
    private int code;
    private AlbumBeanX album;

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public AlbumBeanX getAlbum() {
        return album;
    }

    public void setAlbum(AlbumBeanX album) {
        this.album = album;
    }

    public static class AlbumBeanX {
        private boolean paid;
        private boolean onSale;
        private String commentThreadId;
        private long picId;
        private String briefDesc;
        private long publishTime;
        private String company;
        private ArtistBean artist;
        private String picUrl;
        private int copyrightId;
        private int status;
        private String blurPicUrl;
        private int companyId;
        private long pic;
        private String tags;
        private String description;
        private String subType;
        private String name;
        private int id;
        private String type;
        private int size;
        private InfoBean info;
        private List<SongsBean> songs;
        private List<ArtistsBeanXX> artists;
        private List<String> alias;

        public static class ArtistBean {
            private long img1v1Id;
            private int topicPerson;
            private long picId;
            private String briefDesc;
            private int albumSize;
            private String img1v1Url;
            private String trans;
            private String picUrl;
            private int musicSize;
            private String name;
            private int id;
            private String picId_str;
            private String img1v1Id_str;
            private List<?> alias;
            private List<String> transNames;

        }

        public static class InfoBean {
            private Object latestLikedUsers;
            private boolean liked;
            private Object comments;
            private int resourceType;
            private int resourceId;
            private int commentCount;
            private int shareCount;
            private int likedCount;
            private String threadId;
        }

        public static class SongsBean {

            private boolean starred;
            private double popularity;
            private int starredNum;
            private int playedNum;
            private int dayPlays;
            private int hearTime;
            private String mp3Url;
            private Object rtUrls;
            private int fee;
            private String commentThreadId;
            private HMusicBean hMusic;
            private MMusicBean mMusic;
            private LMusicBean lMusic;
            private AlbumBean album;
            private int mvid;
            private int ftype;
            private int rtype;
            private Object rurl;
            private String copyFrom;
            private int score;
            private int copyrightId;
            private int status;
            private String disc;
            private Object crbt;
            private BMusicBean bMusic;
            private Object rtUrl;
            private int position;
            private int duration;
            private Object audition;
            private Object ringtone;
            private int no;
            private String name;
            private int id;
            private List<ArtistsBeanX> artists;
            private List<?> alias;
            private List<String> transNames;

            public static class HMusicBean {

                private int playTime;
                private int bitrate;
                private long dfsId;
                private int sr;
                private double volumeDelta;
                private Object name;
                private int id;
                private int size;
                private String extension;
                private String dfsId_str;
            }

            public static class MMusicBean {
                private int playTime;
                private int bitrate;
                private long dfsId;
                private int sr;
                private double volumeDelta;
                private Object name;
                private int id;
                private int size;
                private String extension;
            }

            public static class LMusicBean {
                private int playTime;
                private int bitrate;
                private long dfsId;
                private int sr;
                private double volumeDelta;
                private Object name;
                private int id;
                private int size;
                private String extension;
                private String dfsId_str;
            }

            public static class AlbumBean {

                private boolean paid;
                private boolean onSale;
                private String commentThreadId;
                private long picId;
                private String briefDesc;
                private long publishTime;
                private String company;
                private ArtistBeanX artist;
                private String picUrl;
                private int copyrightId;
                private int status;
                private String blurPicUrl;
                private int companyId;
                private long pic;
                private String tags;
                private String description;
                private String subType;
                private String name;
                private int id;
                private String type;
                private int size;
                private List<?> songs;
                private List<ArtistsBean> artists;
                private List<String> alias;

                public static class ArtistBeanX {
                    private long img1v1Id;
                    private int topicPerson;
                    private int picId;
                    private String briefDesc;
                    private int albumSize;
                    private String img1v1Url;
                    private String trans;
                    private String picUrl;
                    private int musicSize;
                    private String name;
                    private int id;
                    private String img1v1Id_str;
                    private List<?> alias;
                }

                public static class ArtistsBean {
                    private long img1v1Id;
                    private int topicPerson;
                    private int picId;
                    private String briefDesc;
                    private int albumSize;
                    private String img1v1Url;
                    private String trans;
                    private String picUrl;
                    private int musicSize;
                    private String name;
                    private int id;
                    private String img1v1Id_str;
                    private List<?> alias;

                }
            }

            public static class BMusicBean {
                private int playTime;
                private int bitrate;
                private long dfsId;
                private int sr;
                private double volumeDelta;
                private Object name;
                private int id;
                private int size;
                private String extension;
                private String dfsId_str;

            }

            public static class ArtistsBeanX {
                private long img1v1Id;
                private int topicPerson;
                private int picId;
                private String briefDesc;
                private int albumSize;
                private String img1v1Url;
                private String trans;
                private String picUrl;
                private int musicSize;
                private String name;
                private int id;
                private String img1v1Id_str;
                private List<?> alias;
            }
        }

        public static class ArtistsBeanXX {
            private long img1v1Id;
            private int topicPerson;
            private int picId;
            private String briefDesc;
            private int albumSize;
            private String img1v1Url;
            private String trans;
            private String picUrl;
            private int musicSize;
            private String name;
            private int id;
            private String img1v1Id_str;
            private List<?> alias;
        }
    }
}

File: src/main/java/freemusic/music/pojo/wy/NeteaseLostSong.kt
package freemusic.music.pojo.wy


data class NeteaseLostSong(
        var code: Int = 0,
        var album: AlbumBeanX? = null
) {
    data class AlbumBeanX(
            var paid: Boolean = false,
            var onSale: Boolean = false,
            var commentThreadId: String? = null,
            var picId: Long = 0,
            var briefDesc: String? = null,
            var publishTime: Long = 0,
            var company: String? = null,
            var artist: ArtistBean? = null,
            var picUrl: String? = null,
            var copyrightId: Int = 0,
            var status: Int = 0,
            var blurPicUrl: String? = null,
            var companyId: Int = 0,
            var pic: Long = 0,
            var tags: String? = null,
            var description: String? = null,
            var subType: String? = null,
            var name: String? = null,
            var id: Int = 0,
            var type: String? = null,
            var size: Int = 0,
            var info: InfoBean? = null,
            var songs: List<SongsBean>? = null,
            var artists: List<ArtistsBeanXX>? = null,
            var alias: List<String>? = null
    ) {
        data class ArtistBean(
                var img1v1Id: Long = 0,
                var topicPerson: Int = 0,
                var picId: Long = 0,
                var briefDesc: String? = null,
                var albumSize: Int = 0,
                var img1v1Url: String? = null,
                var trans: String? = null,
                var picUrl: String? = null,
                var musicSize: Int = 0,
                var name: String? = null,
                var id: Int = 0,
                var picId_str: String? = null,
                var img1v1Id_str: String? = null,
                var alias: List<*>? = null,
                var transNames: List<String>? = null
        )

        data class InfoBean(
                var liked: Boolean = false,
                var comments: Any? = null,
                var resourceType: Int = 0,
                var resourceId: Int = 0,
                var commentCount: Int = 0,
                var shareCount: Int = 0,
                var likedCount: Int = 0,
                var threadId: String? = null
        )

        data class SongsBean(
                var starred: Boolean = false,
                var popularity: Double = 0.toDouble(),
                var starredNum: Int = 0,
                var playedNum: Int = 0,
                var dayPlays: Int = 0,
                var hearTime: Int = 0,
                var mp3Url: String? = null,
                var rtUrls: Any? = null,
                var fee: Int = 0,
                var commentThreadId: String? = null,
                var hMusic: HMusicBean? = null,
                var mMusic: MMusicBean? = null,
                var lMusic: LMusicBean? = null,
                var album: AlbumBean? = null,
                var mvid: Int = 0,
                var ftype: Int = 0,
                var rtype: Int = 0,
                var rurl: Any? = null,
                var copyFrom: String? = null,
                var score: Int = 0,
                var copyrightId: Int = 0,
                var status: Int = 0,
                var disc: String? = null,
                var crbt: Any? = null,
                var bMusic: BMusicBean? = null,
                var rtUrl: Any? = null,
                var position: Int = 0,
                var duration: Int = 0,
                var audition: Any? = null,
                var ringtone: Any? = null,
                var no: Int = 0,
                var name: String? = null,
                var id: Int = 0,
                var artist: List<ArtistsBeanX>? = null,
                var alias: List<*>? = null,
                var transNames: List<String>? = null
        ) {
            data class HMusicBean(
                    var playTime: Int = 0,
                    var bitrate: Int = 0,
                    var dfsId: Long = 0,
                    var sr: Int = 0,
                    var volumeDelta: Double = 0.toDouble(),
                    var name: Any? = null,
                    var id: Int = 0,
                    var size: Int = 0,
                    var extension: String? = null,
                    var dfsId_str: String? = null
            )

            data class MMusicBean(
                    var playTime: Int = 0,
                    var bitrate: Int = 0,
                    var dfsId: Long = 0,
                    var sr: Int = 0,
                    var volumeDelta: Double = 0.toDouble(),
                    var name: Any? = null,
                    var id: Int = 0,
                    var size: Int = 0,
                    var extension: String? = null
            )

            data class LMusicBean(
                    var playTime: Int = 0,
                    var bitrate: Int = 0,
                    var dfsId: Long = 0,
                    var sr: Int = 0,
                    var volumeDelta: Double = 0.toDouble(),
                    var name: Any? = null,
                    var id: Int = 0,
                    var size: Int = 0,
                    var extension: String? = null,
                    var dfsId_str: String? = null
            )

            data class AlbumBean(
                    var paid: Boolean = false,
                    var onSale: Boolean = false,
                    var commentThreadId: String? = null,
                    var picId: Long = 0,
                    var briefDesc: String? = null,
                    var publishTime: Long = 0,
                    var company: String? = null,
                    var artist: ArtistBeanX? = null,
                    var picUrl: String? = null,
                    var copyrightId: Int = 0,
                    var status: Int = 0,
                    var blurPicUrl: String? = null,
                    var companyId: Int = 0,
                    var pic: Long = 0,
                    var tags: String? = null,
                    var description: String? = null,
                    var subType: String? = null,
                    var name: String? = null,
                    var id: Int = 0,
                    var type: String? = null,
                    var size: Int = 0,
                    var songs: List<*>? = null,
                    var artists: List<ArtistsBean>? = null,
                    var alias: List<String>? = null
            ) {
                data class ArtistBeanX(
                        var img1v1Id: Long = 0,
                        var topicPerson: Int = 0,
                        var picId: Int = 0,
                        var briefDesc: String? = null,
                        var albumSize: Int = 0,
                        var img1v1Url: String? = null,
                        var trans: String? = null,
                        var picUrl: String? = null,
                        var musicSize: Int = 0,
                        var name: String? = null,
                        var id: Int = 0,
                        var img1v1Id_st: String,
                        var alias: List<*>? = null
                )

                data class ArtistsBean(
                        var img1v1Id: Long = 0,
                        var topicPerson: Int = 0,
                        var picId: Int = 0,
                        var briefDesc: String? = null,
                        var albumSize: Int = 0,
                        var img1v1Url: String? = null,
                        var trans: String? = null,
                        var picUrl: String? = null,
                        var musicSize: Int = 0,
                        var name: String? = null,
                        var id: Int = 0,
                        var img1v1Id_str: String,
                        var alias: List<*>? = null
                )
            }

            data class BMusicBean(
                    var playTime: Int = 0,
                    var bitrate: Int = 0,
                    var dfsId: Long = 0,
                    var sr: Int = 0,
                    var volumeDelta: Double = 0.toDouble(),
                    var name: Any? = null,
                    var id: Int = 0,
                    var size: Int = 0,
                    var extension: String? = null,
                    var dfsId_str: String? = null
            )

            data class ArtistsBeanX(
                    var img1v1Id: Long = 0,
                    var topicPerson: Int = 0,
                    var picId: Int = 0,
                    var briefDesc: String? = null,
                    var albumSize: Int = 0,
                    var img1v1Url: String? = null,
                    var trans: String? = null,
                    var picUrl: String? = null,
                    var musicSize: Int = 0,
                    var name: String? = null,
                    var id: Int = 0,
                    var img1v1Id_str: String? = null,
                    var alias: List<*>? = null
            )
        }

        data class ArtistsBeanXX(
                var img1v1Id: Long = 0,
                var topicPerson: Int = 0,
                var picId: Int = 0,
                var briefDes: String,
                var albumSize: Int = 0,
                var img1v1Ur: String,
                var trans: String? = null,
                var picUrl: String? = null,
                var musicSize: Int = 0,
                var name: String? = null,
                var id: Int = 0,
                var img1v1Id_str: String,
                var alias: List<*>? = null
        )


    }
}
